src/App.tsx

```diff
@@ -1,10 +1,62 @@
-import React from 'react';
+import React, { ChangeEvent, MouseEvent, useState } from 'react';
+
 import './App.css';
+import { AddTodo } from './components/AddTodo';
+import { StatusRadioButtons } from './components/StatusRadioButtons';
+import { TodoList } from './components/TodoList';
+
+export type Todo = {
+  id : number;
+  comment : string;
+  status : boolean;
+}
 
 function App() {
+
+  //TODO入力欄の状態
+  const [todoText,setTodoText] = useState("");
+  //TODO一覧情報の状態
+  const [todos,setTodos] = useState<Array<Todo>>([]);
+
+  //TODO追加処理
+  const onClickAddTodo = (event : MouseEvent<HTMLButtonElement>) => {
+
+    const comment = (document.getElementById("input-todo-comment") as HTMLInputElement).value;
+
+    //未入力時には追加不可
+    if(comment === "") return;
```

Comment:
この条件は以下のように記載できます。
```suggestion
    if(!comment) return;
```
空文字がfalsyな値であるためです。
https://developer.mozilla.org/ja/docs/Glossary/Falsy

Reply:
レビューありがとうございます。
修正を行いました。